app/contact/api.py

```diff
@@ -0,0 +1,11 @@
+import requests
+from flask import current_app
+
+
+def post_reasons_for_contacting(form=None, payload={}):
+    hostname = current_app.config["CLA_BACKEND_URL"]
+    payload = form.api_payload() if form else payload
+    response = requests.post(
+        url=f"{hostname}/checker/api/v1/reasons_for_contacting/", json=payload
```

Comment:
What do you think of storing the backend url plus `/checker/api/v1` in a config variable? perhaps something like backend_api_blah. As it will be common across all interactions with backend.

Also where do you authenticate with backend?